### DISCUSSION TITLE
请求添加一个能让b站维持旧版或返回旧版的功能

### DISCUSSION THREAD

**Discussion by fengt1an** [the1812/Bilibili-Evolved#4226] [功能建议 / Ideas]
新版的页面真是yts，b站还把返回旧版的按钮藏起来。现在想返回旧版都得F12去改cookie。希望插件集成一下这个功能

**Comment by Geeyun-JY3** [comment]
https://github.com/the1812/Bilibili-Evolved/discussions/4206#discussioncomment-5938148
